src/brevitas/export/onnx/finn/manager.py

```diff
@@ -121,10 +121,9 @@ def export(
                 or args is not None and isinstance(args, QuantTensor))
                 and bool(input_t) != bool(args)):
             
-            if args is None: # Args needs to be defined, but input_t has to be set to None
-                args = input_t
-                input_t = None
-                
+            args = args or input_t # If either one is QuantTensor, args will be QuantTensor
```

Comment:
This isn't specific to QuantTensors though isn't it? It could be any input arguments